db/tests/page_views_tests.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN;
+
+SELECT plan(10);
```

Comment:
What does this do?

Reply:
PGTap let's you assert how many tests will be run, so you can do (psuedocode)

```
select plan(3)

for x of select * from view:
    select is(x, [...])
```

And if the select doesn't return anything, your tests don't pass. Without the `plan(3)` your tests might unexpectedly pass because there's no failing assertion.